org.lamport.tla.toolbox.metainfo.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Jesús Marín <contacto@jesusmg.org> --> 
+<component type="desktop">
+    <id>org.lamport.tla.toolbox</id>
+    <name>TLA+ Toolbox</name>
+    <name xml:lang="es">Herramientas de TLA+</name>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>MIT</project_license>
+    <developer_name>Microsoft Corporation</developer_name>   
+    <developer_name xml:lang="es">Microsoft Corporation</developer_name>   
+    <summary>TLA+ Toolbox is an IDE (integrated development environment) for the TLA+ tools</summary>
+    <summary xml:lang="es">TLA+ Toolbox es un entorno de desarrollo integrado para las herramientas TLA+</summary>
+
+    <content_rating type="oars-1.1" />
+
+    <url type="homepage">https://lamport.azurewebsites.net/tla/tla.html</url>
+    <url type="bugtracker">https://github.com/tlaplus/tlaplus/issues</url>
+    <url type="help">https://groups.google.com/forum/#!forum/tlaplus</url>
+    <url type="contact">https://groups.google.com/forum/#!forum/tlaplus</url>
+  
+    <update_contact>contacto@jesusmg.org</update_contact>
+    <launchable type="desktop-id">org.lamport.tla.toolbox.desktop</launchable>
+
+    <screenshots>
+        <screenshot xml:lang="en" type="default">
+            <image>https://raw.githubusercontent.com/jesmg/org.lamport.tla.toolbox/master/screenshots/screenshot-1.png</image>
+        </screenshot>
+        <screenshot xml:lang="en" type="default">
```

Comment:
```suggestion
        <screenshot>
            <image>https://raw.githubusercontent.com/jesmg/org.lamport.tla.toolbox/master/screenshots/screenshot-1.png</image>
        </screenshot>
        <screenshot>
```

Reply:
"jesmg" in "https://raw.githubusercontent.com/jesmg/org.lamport.tla.toolbox/master/screenshots/screenshot-1.png" looks incorrect.

Reply:
Must change it when Flathub teams creates the definitive repository